### DISCUSSION TITLE
return to drafts link "poof"

### DISCUSSION THREAD

**Discussion by novw** [2Abendsegler/GClh#2188] [Questions] [answered]
After posting a log from drafts the page jumps back to the original cache page. GC puts "return to drafts" on top in the green bar. GClh removes it. Not a big problem, I have "drafts" in my quicklinks, but annoying none the less. This is with "Automatic go back to Drafts after sending to log" turned OFF.

**Comment by capoaira** [comment, marked as answer]
Did you use the config parameter "Try to hide all blue banner to new designed pages"?

**Comment by 2Abendsegler** [comment]
#2216

**Comment by novw** [comment]
Sorry. Missed original reply / question, and am currently abroad.
